point

```diff
@@ -189,14 +197,6 @@ if (program.migrate) {
     return;
 }
 
-// ------------------ Remove Everything ------------ //
-if (program.debug_destroy_everything) {
```

Comment:
This option is something what could be used by some other devs, especially when running the app outside of docker. It's not right to get rid of it without letting anyone know.